src/main/res/values/strings.xml

```diff
@@ -183,21 +183,21 @@
     <string name="installed_applications">Installed</string>
     <string name="sort_filter">Sort and filter</string>
     <string name="new_applications">New apps</string>
-    <string name="install_after_sync">Auto-update</string>
-    <string name="install_after_sync_summary">Install all new versions after looking through repositories for new apps</string>
-    <string name="prefs_updated_apps">Number of apps to show under \"recently updated\"</string>
-    <string name="prefs_new_apps">Number of apps to show under \"new apps\"</string>
+    <string name="install_after_sync">Auto-install</string>
+    <string name="install_after_sync_summary">Install all new versions after looking through repositories for new apps.</string>
+    <string name="prefs_updated_apps">Number of apps to show in \"Recently updated\"</string>
```

Comment:
Particularly want to avoid "update", and make this shorter.
"Fresh"?

Is it new apps only, or all new versions of all apps?